File: 2__C++_Program/7__simple_calculator.cpp
#include <iostream>

using namespace std;

// Main Calculation
int main() {
    // Declare variables and input a,b, symbol
    float a, b, c;
    char symbol;

    cout << "Input first number: "; cin >> a;
    cout << "Input second number: "; cin >> b;
    cout << "Enter +, -, *, or /: "; cin >> symbol;

    // Using if statement with function inside
    if (symbol == '+') {
        cout << "The addition of " << a << " and " << b << " is " << a + b << endl;
    } else if (symbol == '-') {
        cout << "The subtraction of " << a << " and " << b << " is " << a - b << endl;
    } else if (symbol == '*') {
        cout << "The multiplication of " << a << " and " << b << " is " << a * b << endl;
    } else if (symbol == '/') {
        if (b == 0) {
            cout << "Cannot be divided by 0" << endl;
        } else {
            cout << "The division of " << a << " and " << b << " is " << a / b << endl;
        }
    } else {
        cout << "Symbol is undefined" << endl;
    }

    return 0;
}

File: 4__Ruby_Program/7__simple_calculator.rb
# Main Calculator Program

print "Input first number: "
a = gets.chomp.to_f
print "Input second number: "
b = gets.chomp.to_f
print "Enter +, -, *, or /: "
symbol = gets.chomp

if symbol == '+'
    puts "The addition of #{a} and #{b} is #{a + b}"
elsif symbol == '-'
    puts "The subtraction of #{a} and #{b} is #{a - b}"
elsif symbol == '*'
    puts "The multiplication of #{a} and #{b} is #{a * b}"
elsif symbol == '/'
    if b == 0
        puts "Cannot be divided by 0"
    else
        puts "The division of #{a} and #{b} is #{a / b}"
    end
else
    puts "Please use corresponding symbols"
end